setup.py

```diff
@@ -58,7 +58,12 @@
     "Pillow",
     "aicsshparam",
     "pyshtools",
-    "scikit-learn"
+    "scikit-learn",
+    "aicsimageio",
+    "lkaccess",
```

Comment:
This will fail the CI unfortunately. But I think this is why you mentioned last week or someone mentioned that the long term plan is to just switch to "here is my data, CSV or lambda or similar, run it"